Description: Set the values usually set by the ignored spec-version-maven-plugin
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/api/pom.xml
+++ b/api/pom.xml
@@ -137,11 +137,11 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
-                        <Bundle-Version>${spec.bundle.version}</Bundle-Version>
-                        <Bundle-SymbolicName>${spec.bundle.symbolic-name}</Bundle-SymbolicName>
-                        <Extension-Name>${spec.extension.name}</Extension-Name>
-                        <Implementation-Version>${spec.implementation.version}</Implementation-Version>
-                        <Specification-Version>${spec.specification.version}</Specification-Version>
+                        <Bundle-Version>${project.version}</Bundle-Version>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Extension-Name>javax.json</Extension-Name>
+                        <Implementation-Version>${project.version}</Implementation-Version>
+                        <Specification-Version>${project.version}</Specification-Version>
                         <Export-Package>${packages.export}</Export-Package>
                         <Bundle-Description>Java API for JSON Processing (JSON-P) ${spec_version}</Bundle-Description>
                     </instructions>
--- a/impl/pom.xml
+++ b/impl/pom.xml
@@ -115,11 +115,11 @@
                 <extensions>true</extensions>
                 <configuration>
                     <instructions>
-                        <Bundle-Version>${spec.bundle.version}</Bundle-Version>
-                        <Bundle-SymbolicName>${spec.bundle.symbolic-name}</Bundle-SymbolicName>
-                        <Extension-Name>${spec.extension.name}</Extension-Name>
-                        <Implementation-Version>${spec.implementation.version}</Implementation-Version>
-                        <Specification-Version>${spec.specification.version}</Specification-Version>
+                        <Bundle-Version>${project.version}</Bundle-Version>
+                        <Bundle-SymbolicName>org.glassfish.javax.json</Bundle-SymbolicName>
+                        <Extension-Name>javax.json</Extension-Name>
+                        <Implementation-Version>${project.version}</Implementation-Version>
+                        <Specification-Version>1.0</Specification-Version>
                         <Export-Package>${packages.export}</Export-Package>
                         <Private-Package>${packages.private}</Private-Package>
                     </instructions>
